locations/spiders/dessange.py

```
@@ -0,0 +1,31 @@
+    item_attributes = {
+        "brand": "Dessange",
+        "brand_wikidata": "Q62979914",
+    }
+    sitemap_urls = ["https://salon.dessange.com/sitemap.xml"]
+    sitemap_rules = [
+        (r"fr/salon-coiffure/", "parse_sd"),
+    ]
+    time_format = '%H%M'
+    drop_attributes = ["image", "twitter"]
+
+    #website return 404 errors when spidering faster
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)